tests/prepdata/tests/test_data/SA_64x64/landmarks_64x64.csv

```diff
@@ -0,0 +1,180 @@
+Subject,inf insertion point X,inf insertion point Y,sup insertion point X,sup insertion point Y,RV inf X,RV inf Y,Group
```

Comment:
Have you considered to put this under `pykale/data/...`?

Reply:
It was uploaded mistakenly. It is under `pykale/data`.